bob-backend/src/pearl/core.rs

```diff
@@ -101,7 +101,7 @@ impl MetricsProducer for Pearl {
             .iter()
             .chain(once(&self.alien_disk_controller))
             .cloned()
-            .map(|dc| async move { dc.index_memory().await })
+            .map(|dc| async move { dc.active_index_memory().await })
```

Comment:
For metrics we should report full memory occupied by indixes (in active and in inactive blobs)